bio_hansel/main.py

```diff
@@ -67,6 +74,24 @@ def init_parser():
                         type=int,
                         default=200,
                         help='Max k-mer freq/coverage')
+    # Changes
+    parser.add_argument('--low-cov-depth-freq',
+                        type=int,
+                        default=20,
+                        help='Frequencies below this coverage are considered low coverage')
+    parser.add_argument('--missing-total-tiles-max',
```

Comment:
Let's try to have names of things read like regular english statements and sentences, e.g. `--max-missing-tiles`

Is this the threshold for the number of missing positive or negative tiles to consider the result to be untrustworthy (i.e. `qc_status == FAIL`)?

Reply:
This is the maximum percentage of tiles allowed missing before the result is considered an error.

Reply:
Looks like I should change the type from `int` to `float` if it's a percentage.